studio/components/layouts/DatabaseLayout/DatabaseMenu.utils.ts

```diff
@@ -20,6 +20,12 @@ export const generateDatabaseMenu = (
           url: `/project/${ref}/database/triggers`,
           items: [],
         },
+        {
+          name: 'Schemas',
```

Comment:
Can we also change this to be "Schema Visualizer" as well? and we shift it to be below "Tables"